src/bin/edit/draw_menubar.rs

```diff
@@ -81,6 +81,10 @@ fn draw_menu_edit(ctx: &mut Context, state: &mut State) {
         tb.write(ctx.clipboard(), true);
         ctx.needs_rerender();
     }
+    if ctx.menubar_menu_button(loc(LocId::EditSelectAll), 'A', kbmod::CTRL | vk::A) {
+        tb.select_all();
+        ctx.needs_rerender();
+    }
```

Comment:
Could you move this below the Replace block right before `menubar_menu_end`? Afterward I'll merge this PR. Thanks!